draft-ietf-acme-acme.md

```diff
@@ -2497,7 +2515,7 @@ on randomness requirements.
 ~~~~~~~~~~
 {
   "type": "http-01",
-  "url": "https://example.com/acme/authz/0",
+  "url": "https://example.com/acme/authz/eYL3Xw",
```

Comment:
Is there a reason why `eYL3Xw` is much sohrter than the other authz identifiers used in the draft (`PAniVnsZcis`, `r4HqLzrSrpI`)?

Reply:
Search/replace error.  That should have been a challenge URL as well.